tests/commands/run_test.py

```diff
@@ -390,6 +390,38 @@ def test_skip_hook(cap_out, store, repo_with_passing_hook):
         assert msg in printed
 
 
+def test_skip_aliased_hook(cap_out, store, repo_with_passing_hook):
+    with cwd(repo_with_passing_hook):
+        # Add bash hook on there again, aliased
+        with modify_config() as config:
+            config['repos'][0]['hooks'].append(
+                {'id': 'bash_hook', 'alias': 'foo_bash'},
+            )
+        stage_a_file()
```

Comment:
could maybe make an `aliased_repo` fixture and use it in both tests

I think the code is fairly self evident here, I'd drop the comment

Reply:
Done.